app/assets/stylesheets/app/consent_form_preview.scss

```diff
@@ -0,0 +1,38 @@
+.consent-agreement {
+  display: block;
+  background-color: $grey-medium;
+  padding: $gutter;
+  margin-top: $gutter * 2;
+}
+
+.writing-field {
+  display: inline-block;
+  margin-bottom: 24px;
+
+  label {
+    display: block;
+    margin-bottom: $gutter * 1.5;
+    margin-top: $gutter;
+  }
+
+  .space-for-writing {
+    border: 0;
+    border-bottom: 1px solid black;
+  }
+
+  &.name {
+    width: 100%;
+
+    .space-for-writing {
+      width: 97%;
+    }
+  }
+
+  &.signature, &.date {
```

Comment:
49.5 because at 50 the date portions breaks onto a separate line. 95 and 97 to keep things lined up using a relative unit (yes, I'm afraid they were arrived at by trial and error). You're hitting the limits of my CSS here; I never knew how to do this properly with transparently addable properties. Little help?